.github/workflows/on-workflow-call-update-changelog.yml

```
@@ -7,33 +7,38 @@
+      - name: Run Changeset versioning
         run: pnpm changeset version
 
-      - name: Commit and push changelog + bumps
-        run: |
-          git config user.name "github-actions[bot]"
-          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
-          git add .
-          git commit -m "chore: version bump via changesets [skip ci]" || echo "No changes to commit"
-          git push origin main --follow-tags
+      - name: Create Pull Request for changelog update
+        uses: peter-evans/create-pull-request@v5
```

actionlint (1.7.4)

the runner of "peter-evans/create-pull-request@v5" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)